README.md

```diff
@@ -39,8 +46,10 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
     console.log("success!");
   });
 ```
+-The user hash?
```

Comment:
Yes, the convention is that for a POST request the API will return whatever you just posted.